### PR TITLE
Mobile loyout

### PR DESCRIPTION
**Pull request by AlonaLook**

### CODE REVIEW

File: src/js/cards.js [hunk 1]
```diff
@@ -31,10 +31,12 @@ export const buildCards = () => {
 				<div class="card">
 					<div class="card__item card__img" style="background-image: url('./static/images/cards/${image}')"></div>
 					<div class="card__item card__title" name="api">${title}</div>
-					<div class="card__text">
+					<div class="card__description">
```
Comment (PritulaRoman) [inline src/js/cards.js]: Please, fix the intention

File: src/js/cards.js [hunk 2]
```diff
@@ -53,7 +55,9 @@ export const buildCards = () => {
 					<div class="card__title card__title--footer">
 						${title}
 					</div>
-					${footerCardLinks}
+          <div class="wrapper">
```
Comment (PritulaRoman) [inline src/js/cards.js]: Please, fix the intention

File: src/js/cards.js [hunk 3]
```diff
@@ -31,10 +31,12 @@ export const buildCards = () => {
 				<div class="card">
 					<div class="card__item card__img" style="background-image: url('./static/images/cards/${image}')"></div>
 					<div class="card__item card__title" name="api">${title}</div>
-					<div class="card__text">
+					<div class="card__description">
 						${description}
 					</div>
-					${linksCard}
+          <div class="wrapper">
```
Comment (PritulaRoman) [inline src/js/cards.js]: Please, fix the intention

File: src/js/active-search.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import $ from "jquery";
+
+
+const initSearch = () => {
+  $('.js-search-btn').click(function (event) {
+      let x = $('.js-search-btn').parents('.box-bands').addClass('box-bands--active');
+      console.log(x);
```
Comment (PritulaRoman) [inline src/js/active-search.js]: Please, remove the log

File: src/js/active-search.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import $ from "jquery";
+
+
+const initSearch = () => {
+  $('.js-search-btn').click(function (event) {
+      let x = $('.js-search-btn').parents('.box-bands').addClass('box-bands--active');
```
Comment (PritulaRoman) [inline src/js/active-search.js]: Why do we need the variable?

Comment (PritulaRoman) [inline src/js/active-search.js]: The `$('.js-search-btn')` selector used twice. Consider using constant for that.

`const $searchBtn = $('.js-search-btn');`

Comment (PritulaRoman) [reply src/js/active-search.js]: We do not need all `.box-bands` parents. Consider using `.closest('.box-bands')`

File: src/js/cards.js (hunk 1 above)
Comment (AlonaLook) [reply src/js/cards.js]: In which way?

File: src/js/cards.js (hunk 3 above)
Comment (AlonaLook) [reply src/js/cards.js]: In which way?

File: src/js/cards.js (hunk 2 above)
Comment (AlonaLook) [reply src/js/cards.js]: What do you mean?

**Pull request merged**